components/form/RadioButton.vue

```diff
@@ -45,6 +45,10 @@ export default {
     isDisabled() {
       return this.mode === _VIEW || this.disabled;
     },
+
+    isViewChecked() {
+      return this.mode === _VIEW && this.isChecked;
```

Comment:
I think the logic here needs tweaking. In create and edit modes text-muted is applied to all values which makes them look like they're disabled